assets/js/components/user-input/util/constants.js

```diff
@@ -112,3 +157,39 @@ export function getUserInputAnswers() {
 		},
 	};
 }
+
+/**
+ * Gets available answer descriptions for user input settings.
+ *
+ * @since n.e.x.t
+ * @private
+ *
+ * @return {Object} Answer descriptions object.
+ */
+export function getUserInputAnswersDescription() {
+	return {
+		USER_INPUT_ANSWERS_PURPOSE: {
+			sell_products_or_service: __(
+				'E.g. selling products like devices, apparel, equipment, etc or offering services like courses, consulting, tutoring, etc.',
```

Comment:
Etc. should have a full-stop at the end 🤔 

```suggestion
				'E.g. selling products like devices, apparel, equipment, etc. or offering services like courses, consulting, tutoring, etc.',
```